packaging/build_deb

```diff
@@ -9,24 +9,27 @@
 set -ex
 
 readonly SRCDIR="${PWD}"
+readonly TARGET_DIR="${1}"
 readonly WORKDIR="$(mktemp -d --tmpdir=/tmp build_deb.XXXXXX)"
 readonly NAME='libsxg'
 readonly VERSION='0.1'
 readonly TARNAME="${NAME}_${VERSION}"
 readonly TARGET_NAME='libsxg-dev'
-readonly ARCHITECTURE="$(dpkg --print-architecture)"
+readonly ARCH="$(dpkg --print-architecture)"
+readonly RELEASE="$(lsb_release -cs)"
 
 echo "Working in ${WORKDIR}"
 
 git archive --format=tar.gz --output="${WORKDIR}/${TARNAME}.orig.tar.gz" \
-    --prefix="${TARNAME}/" HEAD
+    --prefix="${NAME}/" HEAD
 cd "${WORKDIR}"
 tar xf "${TARNAME}.orig.tar.gz"
-cd "${TARNAME}"
-ln -s "packaging/debian" .
+cd "${NAME}"
+ln -s packaging/debian .
 
-debuild --no-tgz-check -uc -us -b
+debuild -uc -us -b
 
-cp "../${TARGET_NAME}_${VERSION}_${ARCHITECTURE}.deb" "${SRCDIR}"
+cp ../*.deb \
+   "${SRCDIR}/${TARGET_DIR}/${TARGET_NAME}_${VERSION}_${ARCH}_${RELEASE}.deb"
```

Comment:
Why do you need to rename the package?

Well, whatever the reason is, this is really a bad idea. The Debian packaging has the naming convention all packages should obey: `{NAME}_{VER}-{REL}_{ARCH}.deb`; `VER` is the upstream version, `REL` is the Debian release number, `ARCH` is the architecture.

https://www.debian.org/doc/manuals/debian-faq/ch-pkg_basics#s-pkgname

Reply:
OK, Debian version will be contained by directory name.